metrics/web/test/common/presentation/navigation/metrics_page/metrics_page_factory_test.dart

```diff
@@ -2,167 +2,239 @@
+      ".create() returns the dashboard metrics page if the given route name is unknown",
       () {
-        final routeConfiguration = DefaultRoutes.login;
-        final expectedName = routeConfiguration.path;
+        when(routeName.value).thenReturn(unknownRouteName);
 
-        final actualName = metricsPageFactory.create(routeConfiguration).name;
+        final page = metricsPageFactory.create(routeName, null);
 
-        expect(actualName, equals(expectedName));
+        expect(page.child, isA<DashboardPage>());
       },
     );
 
     test(
-      ".create() returns the dashboard metrics page with a name equals to the given route configuration path",
+      ".create() returns the dashboard metrics page with a name equals to the given route name value if the route name is unknown",
       () {
-        final routeConfiguration = DefaultRoutes.dashboard;
-        final expectedName = routeConfiguration.path;
+        when(routeName.value).thenReturn(unknownRouteName);
 
-        final actualName = metricsPageFactory.create(routeConfiguration).name;
+        final page = metricsPageFactory.create(routeName, null);
 
-        expect(actualName, equals(expectedName));
+        expect(page.name, equals(dashboardRouteName.value));
       },
     );
 
     test(
-      ".create() returns the project groups metrics page with a name equals to the given route configuration path",
+      ".create() returns the dashboard metrics page with arguments equals to the given page parameters if the route name is unknown",
       () {
-        final routeConfiguration = DefaultRoutes.projectGroups;
-        final expectedName = routeConfiguration.path;
+        when(routeName.value).thenReturn(unknownRouteName);
 
-        final actualName = metricsPageFactory.create(routeConfiguration).name;
+        final page = metricsPageFactory.create(routeName, pageParameters);
 
-        expect(actualName, equals(expectedName));
+        expect(page.arguments, equals(pageParameters));
       },
     );
 
     test(
-      ".create() returns the debug menu metrics page with a name equals to the given route configuration path",
+      ".create() returns the dashboard metrics page if the given route name is a dashboard",
       () {
-        final routeConfiguration = DefaultRoutes.debugMenu;
-        final expectedName = routeConfiguration.path;
-
-        final actualName = metricsPageFactory.create(routeConfiguration).name;
+        final page = metricsPageFactory.create(dashboardRouteName, null);
 
-        expect(actualName, equals(expectedName));
+        expect(page.child, isA<DashboardPage>());
       },
     );
 
     test(
-      ".create() returns the dashboard metrics page with a name equals to the given route configuration path if the configuration name is unknown",
+      ".create() returns the dashboard metrics page with a name equals to the given route name value",
```

Comment:
✅ 
```suggestion
      ".create() returns the metrics page with a name equals to the given route name value",
```
Also, maybe we need only one such test? Meaning we can test that this method returns the metrics page with the right name on edge cases and then test that this method returns the metrics page with the given route name. What do you think? The same suggestion for `.create() returns the dashboard metrics page with arguments equals to the given page parameters` test.